fog/overseer/server/src/worker.rs

```diff
@@ -511,13 +537,14 @@ where
                             Self::NUMBER_OF_TRIES - current_try as usize;
                         let error_message = match number_of_remaining_tries {
                             0 => format!(
-                                "Did not succeed in setting a new key on node at index {}. Underlying error: {}",
-                                activated_node_index,
+                                "Did not succeed in setting a new key on node {}. Underlying error: {}",
```

Comment:
This message is rather different from the other match arm. Consider consolidating these messages.

Reply:
Done

Reply:
This one still has differing messages.

Reply:
Done